### PR TITLE
Reserve ID for SSRF Scan Rule

### PR DESCRIPTION
**Pull request by ricekot**

### CODE REVIEW

File: docs/scanners.md [hunk 1]
```diff
@@ -172,6 +172,7 @@ Scan rules:
 40043   Log4Shell (CVE-2021-44228, CVE-2021-45046)
 40044   Exponential Entity Expansion (Billion Laughs Attack)
 40045   Spring4Shell (CVE-2022-22965)
+40046   Server Side Request Forgery
```
Comment (ricekot) [inline docs/scanners.md:175]: Should the ID be in the 2XXXX series instead?

Comment (kingthorin) [reply docs/scanners.md:175]: 🤷

**Review by kingthorin — APPROVED** [review]

File: docs/scanners.md (hunk 1 above)
Comment (thc202) [reply docs/scanners.md:175]: The ranges/series no longer indicate anything.

Comment (kingthorin) [reply docs/scanners.md:175]: Should we remove those notes?

**Comment by kingthorin** [comment]
You're right we had cleaned it up already.

**Pull request merged**